src/debugpy/_vendored/pydevd/pydevd_tracing.py

```diff
@@ -244,6 +244,8 @@ def _load_python_helper_lib_uncached():
 
     try:
         filename = get_python_helper_lib_filename()
+        if filename is None:
```

Comment:
This is the actual fix (`None` was returned, and `ctypes.pydll.LoadLibrary(None)` worked afterwards).